src/functions/visualizeBundles.js

```diff
@@ -20,6 +18,10 @@ const visualizeBundles = async ({
   console.log(`\n🖼️   Generating visualization...\n`)
 
   try {
+    // require() of ES Module not supported.
+    const { default: open } = await import('open')
+    const { default: getPort } = await import('get-port')
```

Comment:
newer versions of these packages are written in ES modules. I had updated npm packages in a previous version of this fix, then rolled them back to focus the PR. I've left these in though to make it easier when you do upgrade. They'll surely want to be imported in some better organised way but this got the fix delivered.